packs/src/main/AndroidManifest.xml

```diff
@@ -12,7 +12,7 @@
     <uses-permission android:name="android.permission.READ_SMS" />
     <uses-permission android:name="android.permission.WRITE_SMS" />
 
-    <permission android:name="io.rapidpro.androidchannel.SEND_MESSAGE" android:label="SMS Channel"></permission>
```

Comment:
I think this may be intentionally ambiguous so as to not confused branded versions